### PR TITLE
Don't merge! WIP: temporary hack to disable selinux and run the tests

### PR DESCRIPTION
**Pull request by jimmykarily**
Debugging: https://github.com/kairos-io/kairos/issues/2802

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

**Comment by jimmykarily** [comment]
Relabeling was skipped:

```
kairos@localhost:~$ cat /run/immucore/*.log| less
...
2024-08-27T12:01:15Z WRN (conditional) Skip 'Skipping stage (if statement error: failed to run cat /proc/cmdline | grep "selinux=1"
: exit status 1)' stage name: Relabelling
...
```

File: images/Dockerfile.kairos [hunk 1]
```diff
@@ -10,6 +10,10 @@ RUN ORIG=/usr/bin/grub-editenv; DEST=/usr/sbin/grub2-editenv; [ -e $ORIG ] && [
 # here we add 100Mb aprox
 COPY --from=framework / /
 
+# TODO: temp change to test disabling selinux
+RUN sed -i 's/selinux=1/selinux=0/' /etc/cos/bootargs.cfg
+RUN echo "# edited by dimitris" >> /etc/cos/bootargs.cfg
```
Comment (mauromorales) [inline images/Dockerfile.kairos:15]: I guess this can be removed

Comment (jimmykarily) [reply images/Dockerfile.kairos:15]: Sure, but the whole PR isn't going to be merged. It's just to test the effects of this one quickly: https://github.com/kairos-io/packages/pull/1022

**Comment by jimmykarily** [comment]
The [other PR](https://github.com/kairos-io/packages/pull/1022) is now merged. Closing this.

**Pull request closed without merging**